### ISSUE TITLE
Pandora++ Crashes on IOS 11.0 

### ISSUE THREAD

**Issue by Krytical13** [JMccormick264/PPAppsIPARepo#104]
iPhone 7 Plus, IOS 11.0
Installs, Opens, Allows me to login, then crashes immediately.

Close/Open/Reinstall etc, no dice. just crashes on opening

**Comment by VinZmax** [comment]
Same here but I'm on an iPhone SE running 10.3.1
Used to work and now just crashes after login.
Renewed my license and also used different license and email, but still the same crash.

**Comment by JMccormick264** [comment]
hmm..let me try using the updated one and seeing if that fixes the issue

**Comment by JMccormick264** [comment]
Well then. 10.1 breaks the tweak. "Could not Connect to Pandora"

**Comment by VinZmax** [comment]
Yep, just tried a few versions and I got the same thing now; Could not connect to Pandora.'

**Comment by JascMcWill** [comment]
Has there been any further activity on this? The latest version on Github and IOS11 is showing unable to connect as well. Fingers crossed this can be resolved. I really love using pandora but it has been geoblocked for me and getting rid of ads is a plus.

**Comment by edmundo01** [comment]
Any update in regards to no connection when Pandora is opened using iphone 8 plus version 11?
